sw/host/opentitanlib/src/transport/verilator/subprocess.rs

```diff
@@ -35,9 +35,10 @@ pub struct Subprocess {
 impl Subprocess {
     /// Starts a verilator [`Subprocess`] based on [`Options`].
     pub fn from_options(options: Options) -> Result<Self> {
-        let mut command = Command::new(&options.executable);
+        let mut command = Command::new(String::from("nice"));
```

Comment:
I don't particularly like changing the command from the executable to 'nice', but it's the best way I know to deprioritize this.

Reply:
Does this need the `String::from`? The docs for [Command::new](https://doc.rust-lang.org/std/process/struct.Command.html#method.new) don't use it, but I didn't check the version.

Reply:
I gave it a shot without wrapping in `String::from` and didn't have any trouble compiling. I admit I don't entirely understand the `AsRef<OsStr>` type.

Reply:
yeah, I guess there was some other issue that I've since fixed. Thanks!

Reply:
Are we able to use `setpriority()` or some other POSIX function instead of calling a separate program? I guess `nice` will probably be available anywhere those are, though, so maybe the function is not any better. :thinking: